app/controllers/admin/questions_controller.rb

```diff
@@ -1,17 +1,16 @@
 module Admin
   class QuestionsController < Admin::BaseController
     load_and_authorize_resource :conference, find_by: :short_title
-    load_and_authorize_resource through: :conference, except: [:new, :create]
+    load_and_authorize_resource except: [:create]
 
     def index
       authorize! :index, Question.new(conference_id: @conference.id)
-      @questions = Question.where(global: true).all | Question.where(conference_id: @conference.id)
-      @questions_conference = @conference.questions
-      @new_question = @conference.questions.new
+      @questions = Question.where(global: true).all | Question.where(conference_id: @conference.id) | @conference.questions
```

Comment:
https://github.com/openSUSE/osem/blob/master/app/models/question.rb#L3
From the Question model:
`has_and_belongs_to_many :conferences`
This relationship is used for enabled questions (which will show up in registration)

The conference_id attribute is a manual one, used to know which questions were created for that conference, even if they are not enabled.